sdk/face/azure-ai-vision-face/README.md

```diff
@@ -0,0 +1,470 @@
+
+face_client = FaceClient(endpoint, credential)
+```
+
+## Key concepts
+
+### FaceClient
+
+`FaceClient` provides operations for:
+
+ - Face detection and analysis: Detect human faces in an image and return the rectangle coordinates of their locations,
+   and optinally with landmarks, and face-related attributes. This operation is required as a first step in all the
+   other face recognition scenarios.
+ - Face recognition: Confirm that a user is who they claim to be based on how closely their face data matches the target face.
+   It includes Face verification ("one-to-one" matching) and Face identification ("one-to-many" matching).
+ - Finding similar faces from a smaller set of faces that look similar to the target face.
+ - Grouping faces into several smaller groups based on similarity.
+
+### FaceAdministrationClient
+
+`FaceAdministrationClient` is provided to interact with the following data structures that hold data on faces and
+persons for Face recognition:
+
+ - PersonDirectory
+ - FaceList
+ - LargeFaceList
+ - PersonGroup
+ - LargePersonGroup
+
+### FaceSessionClient
+
+`FaceSessionClient` is provided to interact with sessions which is used for Liveness detection.
+
+ - Create, query, and delete the session.
+ - Query the liveness and verification result.
+ - Query the audit result.
+
+### Long-running operations
+
+Long-running operations are operations which consist of an initial request sent to the service to start an operation,
+followed by polling the service at intervals to determine whether the operation has completed or failed, and if it has
+succeeded, to get the result.
+
+Methods that train a group (LargeFaceList, PersonGroup or LargePersonGroup), create/delete a Person/DynamicPersonGroup, 
+add a face or delete a face from a Person are modeled as long-running operations.
+The client exposes a `begin_<method-name>` method that returns an `LROPoller` or `AsyncLROPoller`. Callers should wait
+for the operation to complete by calling `result()` on the poller object returned from the `begin_<method-name>` method.
+Sample code snippets are provided to illustrate using long-running operations [below](#examples "Examples").
+
+## Examples
+
+The following section provides several code snippets covering some of the most common Face tasks, including:
+
+* [Detecting faces in an image](#face-detection "Face Detection")
+* [Identifing the specific face from a LargePersonGroup](#face-recognition-from-largepersongroup "Face Recognition from LargePersonGroup")
+* [Determining if a face in an video is real (live) or fake (spoof)](#liveness-detection "Liveness Detection")
+
+### Face Detection
+Detect faces and analyze them from an binary data.
+
+```python
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.vision.face import FaceClient
+from azure.ai.vision.face.models import (
+    FaceDetectionModel, FaceRecognitionModel, FaceAttributeTypeDetection03, FaceAttributeTypeRecognition04)
```

Comment:
One is **detection** model which is used to detect faces from an image, and the other is **recognition** model that we'll use it to do face identification or verification (to see if they're the same person or find the person in a group). They're different models for different purposes.

Reply:
Good question. For now, we have two articles explaining the difference between different versions of the models. How about adding these links as comments in this sample?
* Detection model: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/specify-detection-model#evaluate-different-models
* Recognition model: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/specify-recognition-model#model-compatibility

Reply:
I added a few instructions at this commit: 8bd89ff990b0a0f922a7930311f10437d514948e. Hope it is easier to understand now.
![image](https://github.com/Azure/azure-sdk-for-python/assets/100277363/b7d9819b-6688-4fb5-a5aa-12e4fbbeee26)

Reply:
In our documentation, the models are named as `detection_01`, `detection_02`, and `detection_03`. I think it may not be a good idea to rename it in SDK. It may confuse users.

* Detection models: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/specify-detection-model#evaluate-different-models 
![image](https://github.com/Azure/azure-sdk-for-python/assets/100277363/863c74e9-b315-4d91-a685-caeeb67b96ee)

* Recognition models: https://learn.microsoft.com/en-us/azure/ai-services/computer-vision/how-to/specify-recognition-model#detect-faces-with-specified-model
![image](https://github.com/Azure/azure-sdk-for-python/assets/100277363/0fcc7134-667d-4665-8c2d-08b8a9dbb7a3)